corehq/motech/openmrs/const.py

```diff
@@ -21,6 +21,10 @@
     (IMPORT_FREQUENCY_MONTHLY, _('Monthly')),
 )
 
+# device_id for cases added/updated by an OpenmrsImporter.
+# OpenmrsImporter ID is appended to this.
+OPENMRS_IMPORTER_DEVICE_ID = 'openmrs-importer-'
```

Comment:
nit: could change to `OPENMRS_IMPORTER_DEVICE_ID_PREFIX` just to indicate that it's not a complete ID